Tasks/UseDotNetV2/models.ts

```diff
@@ -122,7 +122,7 @@ export class VersionParts {
                 throw tl.loc("OnlyExplicitVersionAllowed", version);
             }
 
-            new semver.Range(version);
+            semver.valid(version);
```

Comment:
what is the difference between Range and valid?

Reply:
semver.valid allows us to validate the version without creating new object. It's more commonly used and more readable as well.

Reply:
@hiyadav  @vineetmimrot  can you please review this PR

Reply:
semver.Range() allows wild characters while doing version checking like 1.2.x. But in this function we are checking for explicit version number and semver.valid() does that. It doesn't allowed wild-characters.

You can further try out the behavior in https://npm.runkit.com/semver

Reply:
@issacnitin : you should check the return value, if it is null then it is invalid entry o/w true